client/components/banner/index.jsx

```diff
@@ -272,7 +271,7 @@ export class Banner extends Component {
 					temporary={ dismissTemporary }
 					onClick={ this.handleDismiss }
 				>
-					{ this.getIcon() }
+					{ ! noIcon && this.getIcon() }
```

Comment:
Hmm on second though 🤔 naming wise, is `showIcon` less weird? We can then provide a default of `true` to avoid needing to update all banner component instances

Reply:
Hm, you're probably right. `showIcon` is less confusing. I'll update this!